tests/core/io/test_image.h

```diff
@@ -304,14 +304,15 @@ TEST_CASE("[Image] Modifying pixels of an image") {
 			image3->get_pixel(1, 0).is_equal_approx(Color(0, 0, 0, 0)),
 			"flip_y() should not leave old pixels behind.");
 
-	// Convert from RGBA to L8, checking alpha
+	// Pre-multiply Alpha then Convert from RGBA to L8, checking alpha
```

Comment:
Is this message still correct?

Reply:
Yep, as he manually premultiplies alpha by calling ``premultiply_alpha()``